org.eclipse.scanning.example/src/org/eclipse/scanning/example/detector/RandomLineModel.java

```diff
@@ -2,7 +2,7 @@
 
 import org.eclipse.scanning.api.device.models.IDetectorModel;
 
-public class RandomLineModel implements IDetectorModel {
+public class RandomLineModel implements IDetectorModel, Cloneable {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "clone()" method to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2157)